styles.css

```diff
@@ -0,0 +1,130 @@
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.container {
+  height: 500px;
+  background: white;
+  width: 375px;
+  border: 1px solid #ccc;
+  box-shadow: 0 4px 10px 0 rgba(204, 77, 77, 0.2);
+  background-repeat: no-repeat;
+  background-image:
+    url(./images/Ellipse\ 8.png),
+    url(./images/Rectangle\ 7.png),
+    url(./images/Ellipse\ 3.png);
+  background-position: center, left, right;
+}
+
+.circle {
+  background-repeat: no-repeat;
+  background-image:
+    url(./images/Ellipse\ 11.png),
+    url(./images/Rectangle\ 5.jpg);
+  background-position: left, center;
+  padding: 1em;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logo {
+  color: #fff;
+  margin: 0 0 0 0.7em;
+  font-weight: 900;
+  cursor: pointer;
+  padding: 0.5em;
+}
+
+.logo-txt {
+  color: rgb(8, 8, 8);
+  text-decoration: none;
+}
+
+.rec {
+  background-image: url(./images/Ellipse\ 15.png);
+}
+
+.menu-image {
+  cursor: pointer;
+  padding: 1.3em;
+}
+
+.intro {
+  width: 350px;
+  height: 100px;
+  left: 16px;
+  top: 106px;
+}
+
+.intro h1 {
+  font-size: 30px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  padding: 20px;
+  color: #1a2236;
+  display: flex;
+  align-items: center;
+}
+
+.summary {
+  width: 440px;
+  height: 170px;
+  left: 16px;
+  top: 100px;
+}
+
+.about-me {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #272c36;
+  margin-left: 20px;
+  margin-top: 30px;
+  margin-right: 10px;
+}
+
+.connect {
+  color: rgb(146, 146, 224);
+  margin-left: 20px;
+  padding: 10px;
+}
+
+.column {
+  float: left;
+  width: 33;
+  padding: 0.2em;
+}
+
+.btn {
+  background-color: #59c0bb;
+  border: none;
+  color: rgb(5, 5, 5);
+  padding: 11px;
+  text-align: center;
+  font-size: 10px;
+  margin: 4px 2px;
+  opacity: 1;
+  transition: 0.3s;
+  border-radius: 10px;
+  text-decoration: none;
+}
+
```

Comment:
- Kindly make sure you implement this requirement for the previous reviewer, here is the [link](https://github.com/AbdimulhinYussuf3675/Mobile-first_portfolio/pull/1#discussion_r978089174) to the requirement 
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) :grinning: :grinning: :grinning: 

## Your design

![image](https://user-images.githubusercontent.com/95297251/191933693-12dfeff2-feaa-40ef-a148-d7bc55d2437f.png)

## Figma design 
!![image](https://user-images.githubusercontent.com/95297251/191933995-c3ca3618-ed26-4571-adff-047cc30f7733.png)